getDataFromDrive/pull_posts.py

```diff
@@ -120,8 +120,12 @@ def export_docx_file_to_markdown(document):
     title = document.get('title')
     with open(f"../docx/{slugify(title)}.docx", "rb") as docx_file:
         result = mammoth.convert_to_markdown(docx_file)
+        fixed_md = result.value
+        fixed_md = re.sub(r"([\w:$-]?)([ \t]*?)__([ \t]*?)([\w:$]+?[\w \t]*?[\w:$,;]+?)([ \t]*?)__([ \t\\-]*?)([\w:$]+?)", r"\1\2\3__\4__\5\6\7", fixed_md)
```

Comment:
Can I even explain how this works? Probably, but....

Reply:
fool tech trivia... convert regex to english!!!